crates/agglayer-jsonrpc-api/src/admin.rs

```diff
@@ -27,8 +30,43 @@ pub enum ProcessNow {
     False,
 }
 
+use serde_with::{serde_as, DisplayFromStr};
+
+/// Token balance entry structure in order to display the balance tree values.
+#[serde_as]
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TokenBalanceEntry {
+    pub origin_network: NetworkId,
+    pub origin_token_address: Address,
+    #[serde_as(as = "DisplayFromStr")]
+    pub amount: U256,
+}
```

Comment:
Wondering if it would make sense to turn this into a doubly nested json dictionary, i.e. `BTreeMap<NetworkId, BTreeMap<Address, U256>>`. Unless there are some compatibility concerns.